src/Xamarin.Android.Build.Tasks/Utilities/Files.cs

```diff
@@ -179,6 +179,35 @@ public static bool CopyIfZipChanged (string source, string destination)
 			return false;
 		}
 
+		public static bool DirectoryIsCaseSensitive (string fullPath)
```

Comment:
This method is creating a new directory that is left behind. But that is the best way to test for a case-sensitive file system, I guess?

I don't think we really need to worry about that, can we just assume it is `true`? It would be fine to give the duplicate file name warning on Linux?

I think it would be bad to have duplicate file names on any platform and the developer should fix the project.

Reply:
I agree I think we should just always check for duplicates

Reply:
This method should at least in theory remove the directory if it creates one when it runs: https://github.com/xamarin/xamarin-android/blob/45c01ff6aa7810a94b6f778e623080588d260e68/src/Xamarin.Android.Build.Tasks/Utilities/Files.cs#L205-L206

But I agree that if we're OK with showing a warning to users who are using asset file names that are technically valid on a case-sensitive file system (when they only differ by capitalization), then I'm in favor of removing `DirectoryIsCaseSensitive()` for the sake of simplicity.

I'll work on that change.

* * *

I also have to take a look at why several of the build tests are failing with:

```
error APT2000: The system cannot find the file specified.
```